src/Auth0.php

```diff
@@ -238,7 +256,8 @@ public function __construct(array $config) {
     $this->refresh_token = $this->store->get("refresh_token");
   }
 
-  public function login($state = null, $connection = null) {
+  public function login($state = null, $connection = null, $additional_params = []) {
```

Comment:
I guess this is NOT a breaking change because you're passing `[]` as default value when the parameter is missing. Am I correct?